src/settings/DataImportSettings.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { Settings } from '@folio/stripes/smart-components';
+import { AppIcon, InfoPopover } from '@folio/stripes/components';
+
+import JobProfiles from './JobProfiles';
+import MatchProfiles from './MatchProfiles';
+import ActionProfiles from './ActionProfiles';
+import FieldMappingProfiles from './FieldMappingProfiles';
+import FileExtensions from './FileExtensions';
+
+import css from './DataImportSettings.css';
+
+class DataImportSettings extends Component {
+  sections = [
+    {
+      label: this.renderProfilesLabel(),
+      pages: [
+        {
+          route: 'job-profiles',
+          label: this.generateSettingsLabel('jobProfiles', 'jobProfiles'),
+          component: JobProfiles,
+        },
+        {
+          route: 'match-profiles',
+          label: this.generateSettingsLabel('matchProfiles', 'matchProfiles'),
+          component: MatchProfiles,
+        },
+        {
+          route: 'action-profiles',
+          label: this.generateSettingsLabel('actionProfiles', 'actionProfiles'),
+          component: ActionProfiles,
+        },
+        {
+          route: 'mapping-profiles',
+          label: this.generateSettingsLabel('fieldMappingProfiles', 'fieldMapping'),
+          component: FieldMappingProfiles,
+        },
+      ],
+    },
+    {
+      label: <FormattedMessage id="ui-data-import.settings.other" />,
+      pages: [
+        {
+          route: 'file-extensions',
+          label: <FormattedMessage id="ui-data-import.settings.fileExtensions" />,
+          component: FileExtensions,
+        },
+      ],
+    },
+  ];
+
+  renderProfilesLabel() {
+    return (
+      <div className={css.profilesLabel}>
+        <FormattedMessage id="ui-data-import.settings.profiles" />
+        <InfoPopover
+          allowAnchorClick
+          buttonHref="https://wiki.folio.org/display/FOLIOtips/Creating+and+using+profiles"
+          buttonLabel={<FormattedMessage id="ui-data-import.settings.learnMore" />}
+          content={<FormattedMessage id="ui-data-import.settings.profilesInfo" />}
+          contentClass={css.profilesPopoverContent}
+          iconSize="medium"
+        />
+      </div>
+    );
+  }
+
+  generateSettingsLabel(labelId, iconKey) {
+    return (
+      <AppIcon
+        size="small"
+        app="data-import"
+        iconKey={iconKey}
+      >
+        <FormattedMessage id={`ui-data-import.settings.${labelId}`} />
+      </AppIcon>
+    );
+  }
+
+  render() {
+    return (
```

Comment:
WHy do we need a return in render function? As I see component is written in Class notation.

Reply:
Please see the documentation for this.
`When called, it should examine this.props and this.state and return one of the following types...`

https://reactjs.org/docs/react-component.html#render